backend/apps/volontulo/views/api.py

```diff
@@ -175,3 +187,20 @@ def contact(request, pk):
             serializer.validated_data,
         )
         return Response(data=True, status=status.HTTP_201_CREATED)
+
+    @staticmethod
+    @detail_route(methods=['GET'], permission_classes=(AllowAny,))
+    # pylint: disable=invalid-name
+    def offers(request, pk):
+        """ Endpoint to get offers for organization """
+        get_object_or_404(Organization, id=pk)
```

Comment:
What do You think about:
```
organization = get_object_or_404(Organization, id=pk)
if logged_as_admin(request):
    offers = organization.offer_set.get_for_administrator()
else:
    offers = organization.offer_set.get_weightened()
```

Reply:
I don't know the details of `get_for_administrator` and `get_weightened`, but I think rather than verifying permission levels in every single query (easy to forget somewhere, harder to refactor, etc.) maybe you could utilize query managers and apply permission filtering there. https://docs.djangoproject.com/en/2.0/topics/db/managers/

Reply:
@magul This works better because the previous version hasn't returned closed offers.

@radoslawroszkowiak These are methods for OffersManager class if I understand it correctly.

What is more important to me right now: for an administrator in organization details view all offers should be returned? for organization owner - also all offers and a possibility to edit them, but for other or unauthenticated users - only published? Who should see rejected offers?